crates/top/rerun_c/src/lib.rs

```diff
@@ -1013,6 +1014,34 @@ pub unsafe extern "C" fn rr_recording_stream_send_columns(
     }
 }
 
+#[allow(unsafe_code)]
```

Comment:
Remove this and hardcode this into the C++ file, with a comment to change it in accordance.